idep/chain.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "idep",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "IDEP",
+  "chain_id": "Antora",
+  "bech32_prefix": "idep",
+  "daemon_name": "idep",
+  "node_home": "$HOME/.ion",
+  "genesis": {
+    "genesis_url": "https://github.com/IDEP-network/Antora/raw/main/genesis.json"
+  },
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
```

Comment:
Should be a new coin type

Reply:
Not at all, IDEP use coin type 118.

Reply:
Wdym by "should be a new coin type"?

Reply:
118 is the coin type for ATOM -- IDEP is not ATOM

Reply:
https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
The majority of blockchains using the Cosmos SDK use the 118 coin type, with the exception of a few. I invite you to look at the other files.

Reply:
@keupsonite i'm happy to discuss further on discord. for hd/bip44 purposes and downstream wallets, it's best to distinguish that it is not cosmos hub.

Reply:
https://github.com/cosmos/chain-registry/blob/master/akash/chain.json#L14
https://github.com/cosmos/chain-registry/blob/master/axelar/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/bitcanna/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/bostrom/chain.json#L14
https://github.com/cosmos/chain-registry/blob/master/carbon/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/cheqd/chain.json#L14
https://github.com/cosmos/chain-registry/blob/master/chronicnetwork/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/comdex/chain.json#L9
https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/crescent/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/decentr/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/dig/chain.json#L15
https://github.com/cosmos/chain-registry/blob/master/emoney/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/fetchhub/chain.json#L14
https://github.com/cosmos/chain-registry/blob/master/genesisl1/chain.json#L15
https://github.com/cosmos/chain-registry/blob/master/gravitybridge/chain.json#L15
https://github.com/cosmos/chain-registry/blob/master/impacthub/chain.json#L18
https://github.com/cosmos/chain-registry/blob/master/irisnet/chain.json#L14
https://github.com/cosmos/chain-registry/blob/master/juno/chain.json#L9
https://github.com/cosmos/chain-registry/blob/master/kichain/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/likecoin/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/meme/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/octa/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/odin/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/oraichain/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/osmosis/chain.json#L18
https://github.com/cosmos/chain-registry/blob/master/rizon/chain.json#L9
https://github.com/cosmos/chain-registry/blob/master/shentu/chain.json#L15
https://github.com/cosmos/chain-registry/blob/master/sommelier/chain.json#L17
https://github.com/cosmos/chain-registry/blob/master/stargaze/chain.json#L11
https://github.com/cosmos/chain-registry/blob/master/umee/chain.json#L11

Reply:
The blockchain is already launched and the team does not intend to change the coin type. Coin type 118 is the best type, as it is natively compatible with Keplr and Ledger. I don't understand your message at all.

Reply:
lol - forget it, we will change coin type later

Reply:
Sorry but I don't understand your point. Maybe the language barrier. So it's ok, you validate?